packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.spec.ts

```diff
@@ -727,4 +727,55 @@ describe('TableColumnNumberText', () => {
             expect(cellView.alignment).toEqual(TextCellViewBaseAlignment.left);
         });
     });
+
+    describe('convertDigitOptionsForDecimalUnitFormat', () => {
+        const convertDigitOptionsTestCases = [
```

Comment:
I believe there is a code path that returns `undefined` as the `maximumFractionDigits`, right? There should be a test case for that.

Reply:
Actually, there isn't anymore. That used to be possible if only a min digits value was provided, but now that the function doesn't take min digits as a parameter, it always returns a number for both properties. I've updated the type accordingly.